packages/addons-v5/test/fixtures.js

```diff
@@ -48,13 +48,13 @@ fixtures.services = {
 }
 
 fixtures.plans = {
-  'heroku-postgresql:hobby-dev': {
+  'heroku-postgresql:no-free': {
```

Comment:
This is minor and a change will cascade into the tests above, but is there a reason we're using a fake plan name like `no-free` here? Would it make sense to use a real plan such as `hobby-basic`? Also, the price has been left at 0 (free), was that intentional?

Reply:
We could also leave this as `hobby-dev` for now and remove it or replace it with a paid addon plan fixture after the free tier is removed in November.

Reply:
That makes more sense @tevanoff . I thought it would make sense since we still have free plans for certain partner addons. If that weren't the case, I would delete that bit in the tests altogether since they're all just displaying `free` under the cli table `Price` heading. Those `no-free` plan names were just to help us distinguish they were not plans we actually support anymore, but checking to make sure `free` still appears when displaying the plans.

Reply:
@zwhitfield3 ah, I see. Thanks for clarifying that. That makes sense!